scripts/env/dev/init.sh

```diff
@@ -52,14 +52,31 @@ check_supported_kernel() {
 
 #function to install/upgrade cortx-py-utils rpm
 install_cortx_py_utils() {
-  #rpm -q cortx-py-utils && yum remove cortx-py-utils -y && yum install cortx-py-utils -y
+  #install cpio
+  if rpm -q 'cpio' ; then
+    echo "cpio already present ... Skipping ..."
+  else
+    yum install cpio -y
+  fi
+
+  # download cortx-py-utils.
+  yumdownloader --destdir="$PWD" cortx-py-utils
+
+  # extract requirements.txt
+  rpm2cpio cortx-py-utils-*.rpm | cpio -idv "./opt/seagate/cortx/utils/conf/requirements.txt"
+
+  # install cortx-py-utils prerequisite
+  pip3 install -r "$PWD/opt/seagate/cortx/utils/conf/requirements.txt" --ignore-installed
```

Comment:
please test this patch on fresh vm .. we earlier saw "gcc moudle not found " issue with this on fresh vm

Reply:
Tested on following fresh VM
`PLAY RECAP *************************************************************************************************************************
127.0.0.1                  : ok=70   changed=62   unreachable=0    failed=0    skipped=6    rescued=0    ignored=0

[root@ssc-vm-3148 cortx-s3server]# git branch
* br/vr/EOS-19433
  main
`